src/test/java/com/ess/pages/MaxNoofCasualLeaveallowedavailforyear.java

```diff
@@ -0,0 +1,137 @@
+package com.ess.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+
+import com.ess.common.BasePage;
+
+public class MaxNoofCasualLeaveallowedavailforyear  extends BasePage {
+	
+	
+
+	@FindBy(id="master")
+	WebElement masterlink;
+
+	@FindBy(xpath="//a[.='Leave Policy']")
+	WebElement LeavePolicy;
+	
+	@FindBy(xpath="(//a[.='Add Leaves'])[1]")
+	WebElement AddLeavesForPolicy;
+	
+
+	@FindBy(xpath="//td[@id='leave_2']//a[contains(text(),'Settings')]")

```

Comment:
@FindBy(xpath="//td[@id='leave_2']//a[contains(text(),'Settings')]")
WebElement settingforleavedefi;
-----Element is not visible check id